Src/BootCamp.Chapter/Statistics.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Xml;
+
+namespace BootCamp.Chapter
+{
+    class Statistics
+    {
+        private decimal highestBalanceEver;
+        private decimal personWithBiggestLoss;
+        private decimal richestPerson;
+        private decimal mostPoorPerson;
+        private string highestBalanceEverName;
+        private string personWithBiggestLossName;
+        private string richestPersonName;
+        private string mostPoorPersonName;
+
+        public Statistics()
+        {
+            highestBalanceEver = 0;
+            personWithBiggestLoss = 0;
+            richestPerson = 0;
+            mostPoorPerson = 0;
+        }
+
+        public void CalculateStats()
+        {
+            UnpackFile fileContents = new UnpackFile();
+            var accountNames = fileContents.ReturnAccounts();
+            for(int i = 0; i < accountNames.Count; i++)
+            {
+                HighestBalanceEver(accountNames, i);
+                PersonWithBiggestLoss(accountNames, i);
+                RichestPerson(accountNames, i);
+                MostPoorPerson(accountNames, i);
+            }
+        }
+
+        public void DisplayStats()
+        {
+            Console.WriteLine($"{highestBalanceEverName}: {ToCurrency(highestBalanceEver)}");
```

Comment:
Move the printing logic (more like string forming logic) to BalanceStatistics object